src/status/chat.nim

```diff
@@ -130,3 +130,9 @@ proc sendMessage*(self: ChatModel, chatId: string, msg: string): string =
   var parsedMessage = parseJson(sentMessage)["result"]["chats"][0]["lastMessage"]
   self.events.emit("messageSent", MsgArgs(message: msg, chatId: chatId, payload: parsedMessage))
   sentMessage
+
+proc markAllChannelMessagesRead*(self: ChatModel, chatId: string): JsonNode =
+  var response = status_chat.markAllRead(chatId)
+  result = parseJson(response)
```

Comment:
As mentioned above, we probably wanna stay consistent with what types we return from `src/status` APIs. Need it to check for error though